SOC/common/simulation.py

```diff
@@ -77,19 +97,17 @@ def AvalancheLoop(self) -> dict:
         :rtype: dict
         """
         self.visited[...] = False
-        self.releases[...] = 0
+        self.releases[...] = 0      # TODO this could definitely simply be overridden in OFC!
```

Comment:
To miejsce wydawało mi się jako jedyne, żeby zerować releases na początku Avalancheloop. Ale przy obecnej jej postaci, to chyba można by rzeczywiście przenieść.

Reply:
Jakby co, to śmiało!:)

Reply:
wlasnie btw ma dependencie na to.. ale to nie jest zbyt potrzebne..

Reply:
Tak, i to jest dobry pomysł, żeby wstawić to tutaj; to co sugeruję to powielić tą metodę w `models.OFC` z dodaniem `self.releases[...] = 0` tylko tam, żeby ten jeden model używał alternatywnej wersji która obsługuje też `releases`.